### PR TITLE
16 include feedback

### PR DESCRIPTION
**Pull request by MFritzN**

### CODE REVIEW

File: src/Representations/ANF.elm [hunk 1]
```diff
@@ -147,3 +150,10 @@ splitANFAndHelp formula =
 sortANFList : List (List String) -> List (List String)
 sortANFList list =
     List.sortBy (\a -> String.fromInt (List.length a) ++ List.foldr (++) "" a) list
+
+
+postProcessANF : String -> String
```
Comment (fabeulous) [inline src/Representations/ANF.elm:155]: Also use this in the Adequacy feature when displaying ANFs.

**Review by fabeulous — COMMENTED** [review]
Only very small comments left

File: src/Representations/ANF.elm [hunk 2]
```diff
@@ -147,3 +150,10 @@ splitANFAndHelp formula =
...
+postProcessANF input =
+    input
+        |> String.replace "∧" "⋅"
```
[the remark this answers is not in the document]
Comment (MFritzN) [reply src/Representations/ANF.elm:158]: Resolbed by cf92b44.

File: src/Representations/ANF.elm (hunk 1 above)
Comment (MFritzN) [reply src/Representations/ANF.elm:155]: Resolved by 7c41533.

File: src/ViewHelpers.elm [hunk 3]
```diff
@@ -29,31 +29,34 @@ syntax =
...
-                [ strong [] [ text "Implication:" ], text " → | \\implies | ->" ]
-            , li []
-                [ strong [] [ text "Exclusive Or:" ], text " ⊕ | \\oplus | ^" ]
-            , li []
-                [ strong [] [ text "Bottom:" ], text " False | \\bot" ]
-            , li []
-                [ strong [] [ text "Top:" ], text " True | \\top" ]
+        , p [] [ text "The tool automatically transforms known symbols into their corresponding Unicode symbols on input. Direct Unicode input is also possible. Here you can find a few of the possible transformations: " ]
+        , table [ class "table" ]
+            [ tr [] [ th [] [ text "Unicode" ], th [] [ text "ASCII" ], th [] [ text "LaTex" ] ]
```
[the remark this answers is not in the document]
Comment (MFritzN) [reply src/ViewHelpers.elm:37]: Resolved by eaf6c25.

**Pull request merged**